src/lib/tecs/TECS.hpp

```diff
@@ -658,6 +662,15 @@ class TECS
 	void initControlParams(float target_climbrate, float target_sinkrate, float eas_to_tas, float pitch_limit_max,
 			       float pitch_limit_min, float throttle_min, float throttle_setpoint_max, float throttle_trim);
 
+	/**
+	 * @brief calculate true airspeed setpoint
```

Comment:
could drop a comment here what the calculation is based on (for now only fast descend ratio)